drivers/gles3/rasterizer_canvas_gles3.cpp

```diff
@@ -2057,7 +2057,17 @@ void RasterizerCanvasGLES3::set_shadow_texture_size(int p_size) {
 		WARN_PRINT("Attempting to set CanvasItem shadow atlas size to " + itos(p_size) + " which is beyond limit of " + itos(config->max_texture_size) + "supported by hardware.");
```

Comment:
Maybe should be a `return` here if you are going to be changing the buffers below?

Reply:
Ah, yep good point, it should be fine. :+1: